builders/interoperability/xcm/xc20/xtokens.md

```diff
@@ -5,7 +5,7 @@ description: Learn how to send XC-20s to other chains using the X-Tokens pallet.
 
 # Using the X-Tokens Pallet To Send XC-20s
 
-![X-Tokens Precompile Contracts Banner](/images/builders/xcm/xc20/xtokens/xtokens-banner.png)
+![X-Tokens Precompile Contracts Banner](/images/builders/interoperability/xcm/xc20/xtokens/xtokens-banner.png)
 
 ## Introduction {: #introduction } 
 
```

Comment:
Outside of scope of review but while we're at it, line 35 has typo here "curriencies"

Reply:
good catch!